app/api-requests/rpa-api/base.js

```diff
@@ -1,11 +1,13 @@
 const wreck = require('@hapi/wreck')
 const session = require('../../session')
 const { tokens } = require('../../session/keys')
+const config = require('../../config')
 
 const get = async (hostname, url, request, headers = {}) => {
   const token = session.getToken(request, tokens.accessToken)
-  headers.Authorization = token
-  console.log(`${new Date().toISOString()} Request Message to RPA: ${JSON.stringify(`${hostname}${url}`)}`)
+  headers['X-Forwarded-Authorization'] = token
+  headers['Ocp-Apim-Subscription-Key'] = config.authConfig.ruralPaymentsAgency.ocpApimSubscriptionKey

```

Comment:
could we put this key into the apim config section?

Reply:
Code updated - config value moved into the apim section